README.rst

```diff
@@ -11,58 +11,41 @@ scripts.
 Inspektor knows how to:
 
 1) Check code with the help of `pylint`.
-2) Check indentation of your code, correcting it if necessary.
-3) Check whether your code is PEP8 compliant, correcting it if necessary.
-4) Run unittests inside your tree, provided they follow some conventions.
-5) If your project is hosted on the autotest github area, it can apply pull
+2) Check indentation of your code with the help of `pycodestyle`,
+   correcting it if you so deem appropriate.
+3) Check whether your code is PEP8 compliant, correcting it if necessary
+   (only works if you have `autopep8 installed) if you so deem appropriate.
+4) If your project is hosted on the autotest github area, it can apply pull
    requests made against it, and check if it introduced any regression from
-   the metrics outlined above
+   the metrics outlined above.
 
 This all assumes you're working on a version control checkout of your code.
 Currently inspektor knows how to handle subversion and git.
 
 Installing inspektor
 --------------------
 
-If there are no packages for your distro, you'll have to resort to good old:
+If there are no packages for your distro, you'll have to use pip:
```

Comment:
Based on the later recommendations I'd change this sentence to emphasize that `pip` is actually the (only?) supported but definitely recommended method to install inspektor, rather than to say you should use it unless there is other way.